### PR TITLE
Final submission - Group 2

### PR DESCRIPTION
**Pull request by priyadesh99**
Only works when dest is within radius

### CODE REVIEW

File: players/default_player.py [hunk 1]
```diff
@@ -39,6 +39,100 @@ def __init__(self, rng: np.random.Generator, logger: logging.Logger,
         self.logger = logger
         self.maximum_door_frequency = maximum_door_frequency
         self.radius = radius
+        # x, y in seens and knowns is centered around start x, y
```
Comment (divyang-mittal) [inline players/default_player.py]: Please remove these changes from default player

File: timing_maze_game.py [hunk 2]
```diff
@@ -175,12 +175,10 @@ def initialize(self, maze):
             # Generate a frequency for each cell between 0 and max_door_frequency using the rng
             # self.logger.info("Generating random maze using seed {}".format(self.rng.bit_generator.seed))
             while 1:
-                self.cur_pos = np.array([self.rng.integers(0, constants.map_dim),
-                                         self.rng.integers(0, constants.map_dim)])
+                self.cur_pos = np.array([45,50])
```
Comment (divyang-mittal) [inline timing_maze_game.py]: Please revert this

File: timing_maze_game.py [hunk 3]
```diff
@@ -175,12 +175,10 @@ def initialize(self, maze):
...
                 self.start_pos = self.cur_pos.copy()
                 while 1:
-                    self.end_pos = np.array([self.rng.integers(0, constants.map_dim),
-                                             self.rng.integers(0, constants.map_dim)])
+                    self.end_pos = np.array([51,51])
```
Comment (divyang-mittal) [inline timing_maze_game.py]: Please revert this

File: players/default_player.py (hunk 1 above)
Comment (priyadesh99) [reply players/default_player.py]: Done

File: timing_maze_game.py (hunk 2 above)
Comment (priyadesh99) [reply timing_maze_game.py]: Done

File: timing_maze_game.py (hunk 3 above)
Comment (priyadesh99) [reply timing_maze_game.py]: Done

**Review by divyang-mittal — APPROVED** [review]

**Pull request merged**